ocs_ci/ocs/node.py

```diff
@@ -1222,16 +1223,17 @@ def taint_nodes(nodes, taint_label=constants.OCS_TAINT):
 
     """
     ocp_obj = ocp.OCP()
+    taint_label = taint_label if taint_label else constants.OPERATOR_NODE_TAINT
     for node in nodes:
         command = f"adm taint node {node} {taint_label}"
         try:
             ocp_obj.exec_oc_cmd(command)
-            logging.info(f"Successfully tainted {node} with OCS storage taint")
+            logging.info(f"Successfully tainted {node} with taint {taint_label}")
         except Exception as e:
             logging.info(f"{node} was not tainted - {e}")
 
 
-def check_taint_on_ocs_nodes(taint=constants.OPERATOR_NODE_TAINT):
+def check_taint_on_nodes(taint=None):
```

Comment:
Change None to False as you have this value as bool, so I would keep it this way as False here

Reply:
Func is to check taint on nodes, so passing taint is necessary here this cannot be a bool value

Reply:
sorry, I saw bool but it was in return section as I didn't open whole block of code I got confused.. Ignore this.